media/js/src/wizard.js

```diff
@@ -1,15 +1,52 @@
 ;(function() {
 
+    var allRadioButtonsSelected = function(radios) {
+        var groups = {};
+        radios.each(function(a) {
+            groups[$(this).attr('name')] = 1;
+        });
+        var total = Object.keys(groups).length;
+        var selected = radios.filter(function(index) {
+            return $(this).is(':checked');
+        });
+        return total == selected.size();
```

Comment:
By default, jshint only catches use of `==` when comparing with `null`: http://jshint.com/docs/options/#eqnull

We could also add this option to our defaults: http://jshint.com/docs/options/#eqeqeq